react/features/chat/components/web/ChatInput.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable linebreak-style */
```

Comment:
Don't do this please. Configure your git client or ediitor to use the appropriate line breaks instead.

Reply:
got it, Must I recorrect and commit again?

Reply:
![image](https://github.com/jitsi/jitsi-meet/assets/135811767/c81c2a88-ec4b-4764-820b-7fb48f0e5e6e)

Reply:
![image](https://github.com/jitsi/jitsi-meet/assets/135811767/a6181217-f8fe-40dd-9889-350a8c67685f)